### DISCUSSION TITLE
OWASP Machine Learning Security Top 10 - Draft release v0.2 - ML01:2023 Input Manipulation Attack

### DISCUSSION THREAD

**Discussion by ankitloud** [OWASP/www-project-machine-learning-security-top-10#122] [Announcements]
### Discussed in https://github.com/OWASP/www-project-machine-learning-security-top-10/discussions/115

<div type='discussions-op-text'>

<sup>Originally posted by **giscus[bot]** September 20, 2023</sup>
# OWASP Machine Learning Security Top 10 - Draft release v0.2 - ML01:2023 Input Manipulation Attack

https://mltop10.info/ML01_2023-Input_Manipulation_Attack.html

</div>

**Comment by ankitloud** [comment]
Attack vector Exploitability: 5 (Easy) doesn't look so easy. Since attack requires technical knowledge of deep learning and image processing techniques. How someone first get into the data source and get the data and then exploit. In the same way impact should be high rather difficult since if data is manipulated then it can generate lot of unexpected results and few of them could be life threating as well.

**Comment by shsingh** [comment]
duplicate of #115

**Reply by ankitloud** [reply]
what is the outcome of this ticket, I can see 115 become dup of 122 and then 123 but didn't see any resolution comment here ?
